<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainUI</class>
  <widget class="QWidget" name="MainUI">
    <property name="geometry">
      <rect>
	<x>0</x>
	<y>0</y>
	<width>659</width>
	<height>405</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <property name="windowIcon">
      <iconset theme="fcitx-rime">
	<normaloff/>
      </iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
	<widget class="QTabWidget" name="frcu_tab">
	  <widget class="QWidget" name="general_tab">
	    <attribute name="title">
	      <string>General</string>
	    </attribute>
	    <layout class="QVBoxLayout" name="verticallayout_general">
	      <item>
		<layout class="QHBoxLayout" name="horizontalLayout_toggle">
		  <item>
		    <widget class="QLabel" name="toggle_label">
		      <property name="text"><string>Toggle Shortcut</string></property>
		    </widget>
		  </item>
		  <item>
		    <widget class="FcitxQtKeySequenceWidget" name="toggle_shortcut">
		    </widget>
		  </item>
		  <item>
		    <widget class="FcitxQtKeySequenceWidget" name="toggle_shortcut_2">
		    </widget>
		  </item>
		</layout>
	      </item>
	      <item>
		<layout class="QHBoxLayout" name="horizontalLayout_toggle1">
		  <item>
		    <widget class="QLabel" name="candidate_word_number_label">
		      <property name="text"><string>Candidate Word Number</string></property>
		    </widget>
		  </item>
		  <item>
		     <widget class="QSpinBox" name="cand_cnt_spinbox" >
		       <property name="objectName" >
			 <string notr="true" >CandidateWordNumber</string>
		       </property>
		     </widget>
		  </item>
		</layout>
	      </item>
	    </layout>
	  </widget>
	  <widget class="QWidget" name="luna_pinyin_tab">
	    <attribute name="title">
	      <string>朙月拼音</string>
	    </attribute>
	  </widget>
	  <widget class="QWidget" name="=luna_pinyin_simp_tab">
	    <attribute name="title">
	      <string>朙月拼音·简化字</string>
	    </attribute>
	  </widget>
	</widget>
      </item>
      <item>
      	<spacer name="horizontalSpacer">
      	  <property name="orientation">
      	    <enum>Qt::Horizontal</enum>
      	  </property>
      	  <property name="sizeHint" stdset="0">
      	    <size>
      	      <width>40</width>
      	      <height>20</height>
      	    </size>
      	  </property>
      	</spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>FcitxQtKeySequenceWidget</class>
      <extends>QWidget</extends>
      <header>fcitxqtkeysequencewidget.h</header>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
